crates/matrix-sdk/Cargo.toml

```diff
@@ -55,6 +55,9 @@ docsrs = [
     "image-proc",
 ]
 
+# This is an internal feature only used for tests
+__test = []
```

Comment:
To be honest, I don't like this at all, I want `cargo test` to continue on working. Can't we just leave the single test that uses `homeserver_from_name()` where it is?